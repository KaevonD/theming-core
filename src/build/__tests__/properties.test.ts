// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
import { preset, defaultsResolution } from '../../__fixtures__/common';
import { calculatePropertiesMap } from '../properties';

test('generates map from theme and variables', () => {
  const map = calculatePropertiesMap(defaultsResolution, preset.variablesMap);

  expect(map).toMatchSnapshot();
});
