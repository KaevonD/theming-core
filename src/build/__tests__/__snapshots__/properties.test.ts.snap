// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates map from theme and variables 1`] = `
Object {
  "appear": "--appear-var-bqka5u",
  "black": "--black-var-r5u19n",
  "boxShadow": "--boxShadow-var-tmv0rp",
  "brown": "--brown-var-yg13cv",
  "buttonShadow": "--buttonShadow-var-b951yn",
  "containerShadowBase": "--containerShadowBase-var-imnmlp",
  "fontFamilyBase": "--fontFamilyBase-var-omo89z",
  "fontFamilyBody": "--fontFamilyBody-var-jm19vz",
  "grey": "--grey-var-b6v5yp",
  "lineShadow": "--lineShadow-var-kcqn0o",
  "medium": "--medium-var-tveajv",
  "modalShadowContainer": "--modalShadowContainer-var-x6sehp",
  "scaledSize": "--scaledSize-var-zvg51q",
  "shadow": "--shadow-var-a2m1xh",
  "small": "--small-var-87qott",
}
`;
